tests/snappy_pipeline/workflows/test_workflows_adapter_trimming.py

```diff
@@ -0,0 +1,181 @@
+# -*- coding: utf-8 -*-
+"""Tests for the hla_typing workflow module code"""
+
+import textwrap
+
+import pytest
+import ruamel.yaml as ruamel_yaml
+from snakemake.io import Wildcards
+
+from snappy_pipeline.workflows.adapter_trimming import AdapterTrimmingWorkflow
+
+from .conftest import patch_module_fs
+
+__author__ = "Eric Blanc <eric.blanc@bih-charite.de>"
+
+
+@pytest.fixture(scope="module")  # otherwise: performance issues
+def minimal_config():
+    """Return YAML parsing result for (germline) configuration"""
+    yaml = ruamel_yaml.YAML()
+    return yaml.load(
+        textwrap.dedent(
+            r"""
+        static_data_config:
+          reference:
+            path: /path/to/ref.fa
+
```

Comment:
By default, it will only run `bbduk` (`DEFAULT_CONFIG`). I would include both tools in the test config.

Reply:
...and I would include a test for the `BaseStep`. Are all the `get_results` as expected?